test/plots/wheelPlotTests.ts

```diff
@@ -0,0 +1,296 @@
+        assert.isUndefined(wheelPlot.innerRadius(), "innerRadius is initiized to undefined");
+
+        wheelPlot.innerRadius(0);
+        wheelPlot.renderTo(svg);
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[0], 0, dataset), 0, "access innerRadius that is set to a constant");
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[1], 1, dataset), 0, "access innerRadius that is set to a constant");
+        assert.isUndefined(wheelPlot.innerRadius().scale, "scale of innerRadius is undefined");
+
+        wheelPlot.innerRadius((d: any) => d.r1);
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[0], 0, dataset), 0, "access innerRadius correctly without a scale");
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[1], 1, dataset), 1, "access innerRadius correctly without a scale");
+        assert.isUndefined(wheelPlot.innerRadius().scale, "scale of innerRadius is undefined");
+
+        wheelPlot.innerRadius((d: any) => d.r1, rScale);
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[0], 0, dataset), 0, "access innerRadius correctly with a scale");
+        assert.strictEqual(wheelPlot.innerRadius().accessor(data[1], 1, dataset), 1, "access innerRadius correctly with a scale");
+        assert.deepEqual(wheelPlot.innerRadius().scale, rScale, "scale of innerRadius is set correctly");
+        svg.remove();
+      });
+
+      it("can set and get outerRadius", () => {
+        wheelPlot.innerRadius((d) => d.r1);
+        assert.isUndefined(wheelPlot.outerRadius(), "outerRadius is initiized to undefined");
+
+        wheelPlot.outerRadius(0);
+        wheelPlot.renderTo(svg);
+        assert.strictEqual(wheelPlot.outerRadius().accessor(data[0], 0, dataset), 0, "access outerRadius that is set to a constant");
+        assert.strictEqual(wheelPlot.outerRadius().accessor(data[1], 1, dataset), 0, "access outerRadius that is set to a constant");
+        assert.isUndefined(wheelPlot.outerRadius().scale, "scale of outerRadius is undefined");
+
+        wheelPlot.outerRadius((d: any) => d.r2);
+        assert.strictEqual(wheelPlot.outerRadius().accessor(data[0], 0, dataset), 1, "access outerRadius correctly without a scale");
+        assert.strictEqual(wheelPlot.outerRadius().accessor(data[1], 1, dataset), 2, "access outerRadius correctly without a scale");
+        assert.isUndefined(wheelPlot.outerRadius().scale, "scale of outerRadius is undefined");
+
+        svg.remove();
+      });
+
+      it("sets scale of outerRadius when scale of innerRadius is set", () => {
+        wheelPlot.outerRadius((d) => d.r2);
+        wheelPlot.renderTo(svg);
+
+        assert.isUndefined(wheelPlot.outerRadius().scale, "scale of outerRadius is undefined initially");
+
+        wheelPlot.innerRadius((d) => d.r1, rScale);
+        assert.deepEqual(wheelPlot.outerRadius().scale, rScale, "scale of outerRadius is set to be the same scale as innerRadius");
```

Comment:
Might be worth confirming that setting `outerRadius()` after setting `innerRadius()` still picks up the `Scale`.